ci.sh

```diff
@@ -249,6 +252,22 @@ if [ "$PROVIDER_NAME" = "all" ]; then
     popd
 fi
 
+# Test the SQLite KIM
```

Comment:
Good point, I think instead of having extra logic we could just change the path for the config file used, but apart from that simply run as if `all` was requested.

Reply:
Oh, actually no, because the `all` flow does some tests (like the ones I mentioned in reply to your no. 2 question above) which aren't inteded for the SQLite KIM, so we need it to be separate. I could maybe just set a different flag and branch around the stuff that we don't want to run from `all`, but it'd be just as messy.